pkg/lwip/contrib/_netif.c

```diff
@@ -32,4 +33,53 @@ int netif_get_name(netif_t *iface, char *name)
     return res;
 }
 
+int netif_get_opt(netif_t *iface, netopt_t opt, uint16_t context,
+                  void *value, size_t max_len)
+{
+    lwip_netif_t *lwip_netif = (lwip_netif_t*) iface;
```

Comment:
```suggestion
    (void)context;
    lwip_netif_t *lwip_netif = (lwip_netif_t*) iface;
```
still an unused variable left